frontend/src/features/toy/ToyList.tsx

```diff
@@ -1,42 +1,47 @@
-import { useEffect, useState } from "react";
-import { BASE_URL } from "../../utils/consts";
+/* eslint-disable no-underscore-dangle */
+import { useEffect } from "react";
+import { useAppDispatch, useAppSelector } from "../../hooks/redux";
+import { RootState } from "../../store/store";
+import { fetchToys } from "./toySlice";
 import ToyCard from "./ToyCard";
-import { Toy } from "./toyTypes";
 
 function ToyList() {
-  const [toys, setToys] = useState<Toy[]>([]);
-  useEffect(() => {
-    async function fetchToys() {
-      try {
-        const response = await fetch(`${BASE_URL}/toys`);
-        const data = await response.json();
-        const toysArr: Toy[] = data.data;
+  const dispatch = useAppDispatch();
+  const toys = useAppSelector((state: RootState) => state.toys.toys);
```

Comment:
that's ok, but it can also be shorter because we use hooks that provide us with typing from redux
also here you can use destructuring
```
 const { error } = useAppSelector((state) => state.toys);
  ```